brd-ios/breadwallet/src/Helpers/PaddedTextField.swift

```diff
@@ -5,7 +5,7 @@
 import UIKit
 
 class PaddedTextField: UITextField {
-    let padding = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
+    let padding = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 48)
```

Comment:
this makes all the text fields have 48 right padding. that means the text will be cut off way before it reaches the text field bounds. is it being handled anywhere?